subprojects/docs/src/docs/userguide/signingPlugin.adoc

```diff
@@ -195,16 +197,24 @@ For a task to be “signable”, it must produce an archive of some type. Tasks
 [[sec:conditional_signing]]
 ==== Conditional Signing
 
-A common usage pattern is to only sign build artifacts under certain conditions. For example, you may not wish to sign artifacts for non-release versions. To achieve this, you can specify that signing is only required under certain conditions.
+A common usage pattern is to only require signing build artifacts under certain conditions. For example, you may not need to sign artifacts for non-release versions. To achieve this, you can specify that signing is only required under certain conditions.
```

Comment:
I think this needs some extra clarity. I prefer:

    A common usage pattern is to require the signing of build artifacts only under certain conditions.

Shifting the 'only' emphasises that it applies to the "certain conditions". This is the only change I think has to be made, but I have some other comments.

The last sentence also feels like a repetition of the first. I know it's not part of your change, but I would consider something like:

    To achieve this, you can specify the condition as an argument of the `required()` method.

As a general FYI — and again, I realise this isn't related to your changes — it's generally a good idea to explain what an example is for or what it's showing _before_ the example, not just after. I tend to leave the trailing paragraphs for a more detailed breakdown of an example.

Reply:
Makes sense!